Source/ASDisplayNode+Layout.mm

```diff
@@ -305,24 +305,22 @@ - (void)_u_measureNodeWithBoundsIfNecessary:(CGRect)bounds
   }
   
   CGSize boundsSizeForLayout = ASCeilSizeValues(bounds.size);
-  NSUInteger calculatedVersion = _calculatedDisplayNodeLayout->version;
```

Comment:
Note to reviewers: No change in the logic of this method (`-_u_measureNodeWithBoundsIfNecessary:`), just call the new `isValid()` function to (hopefully) increase readability.